core/app/config_test.go

```diff
@@ -1 +1,45 @@
 package app
+
```

Comment:
Remove the comments since we should not push commented function code

Reply:
Done - I uncommented the code, it was actually a valid test, I commented it out when I was debugging something else